tests/gis_tests/geoapp/test_sitemaps.py

```diff
@@ -60,10 +61,10 @@ def test_geositemap_kml(self):
                 elif kml_type == 'kmz':
                     # Have to decompress KMZ before parsing.
                     buf = BytesIO(self.client.get(kml_url).content)
-                    zf = zipfile.ZipFile(buf)
-                    self.assertEqual(1, len(zf.filelist))
-                    self.assertEqual('doc.kml', zf.filelist[0].filename)
-                    kml_doc = minidom.parseString(zf.read('doc.kml'))
+                    with contextlib.closing(zipfile.ZipFile(buf)) as zf:
```

Comment:
ZipFile already supports the context management protocol, so

```py
with zipfile.ZipFile(buf) as zf:
    ...
```

would be better.